From 024f1b6dc08daf4272177a2cfc30c73af476b13d Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 24 Sep 2013 13:54:08 +0100
Subject: [PATCH 044/117] sparsify: Fix incorrect reference to documentation
 from tool.

(cherry picked from commit ae2dd1a9e77b06ae9e7fc8ff47d8b5ea7331e0a9)
---
 sparsify/sparsify.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sparsify/sparsify.ml b/sparsify/sparsify.ml
index 0364103..758378f 100644
--- a/sparsify/sparsify.ml
+++ b/sparsify/sparsify.ml
@@ -264,7 +264,7 @@ Note this is an overestimate.  If the guest disk is full of data
 then not as much free space would be required.
 
 You can ignore this warning or change it to a hard failure using the
---check-tmpdir=(ignore|continue|warn|fail) option.  See virt-sysprep(1).
+--check-tmpdir=(ignore|continue|warn|fail) option.  See virt-sparsify(1).
 
 %!")
         tmpdir (human_size virtual_size)
-- 
2.5.0

